packages/core/src/test/shared/wizards/wizard.test.ts

```diff
@@ -211,11 +211,11 @@ describe('Wizard', function () {
 
     it('binds prompter to (sync AND async) property', async function () {
         wizard.form.prop1.bindPrompter(() => helloPrompter)
-        wizard.form.prop3.bindPrompter(async () => new SkipPrompter('helloooo (async)'))
+        wizard.form.prop3.bindPrompter(async () => new SkipPrompter())
 
         const result = await wizard.run()
         assert.strictEqual(result?.prop1, 'hello')
-        assert.strictEqual(result?.prop3, 'helloooo (async)')
+        assert(!result?.prop3)
```

Comment:
Can we add a new test to verify the new functionality?

Reply:
Added test for NestedWizard.